packages/fluentui/react-builder/package.json

```diff
@@ -9,6 +9,7 @@
     "@fluentui/react-component-event-listener": "^0.51.0",
     "@fluentui/react-icons-northstar": "^0.51.0",
     "@fluentui/react-northstar": "^0.51.0",
+    "@fluentui/code-sandbox": "^0.51.0",
```

Comment:
```suggestion
    "@fluentui/docs-components": "^0.51.0",
```

?

Reply:
docs-components are on line 8 :)

Reply:
I see... And we need it only to get a version... I don't have a better proposal now.